pom.xml

```diff
@@ -102,6 +103,22 @@
             </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <requireUpperBoundDeps>
+              <excludes combine.children="append">
+                <!-- promoted-builds requests 1.9.2, core requests 1.8.4 -->
+                <exclude>org.apache.ant:ant</exclude>
```

Comment:
Why would `promoted-builds` depend on a different version of `ant`? Seems like a mistake in that plugin. Still true with new versions?

Better to ignore that dependency trail using `<exclusion>`.